xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/javascript.vm

```diff
@@ -271,11 +271,10 @@ $xwiki.jsfx.use("flamingo$jsExtension", {'forceSkinAction' : true, 'language' :
         #set($searchSuggestConfigHash = "${searchSuggestConfigHash}|${sourceId}")
       #end
     #end
-    $xwiki.jsfx.use('uicomponents/search/searchSuggest.js', {
-      'forceSkinAction' : true,
-      'h' : $searchSuggestConfigHash.hashCode()
-    })##
-    $xwiki.ssfx.use('uicomponents/search/searchSuggest.css', true)##
+    <script src="${services.webjars.url('org.xwiki.platform:xwiki-platform-search-webjar', 'searchSuggest.min.js', {'evaluate': true, 'h': $searchSuggestConfigHash.hashCode()})}" defer="defer"></script>
```

Comment:
Would be nice to use ``${jsExtension}`` (see above), so that we can debug more easily (sometimes the source map is not very helpful). Also, the line is too long.

Reply:
Good point, I missed that.